static/js/vue-cdr-access/tests/unit/permissionUtils.spec.js

```diff
@@ -0,0 +1,147 @@
+        ],
+        counts: {
+            child: 5
+        },
+        created: 917049600000,
+        title: "testAdminUnit",
+        type: "AdminUnit",
+        contentStatus: [
+            "Described"
+        ],
+        rollup: "353ee09f-a4ed-461e-a436-18a1bee77b01",
+        objectPath: [
+            {
+                pid: "collections",
+                name: "Content Collections Root",
+                container: true
+            },
+            {
+                pid: "353ee09f-a4ed-461e-a436-18a1bee77b01",
+                name: "testAdminUnit",
+                container: true
+            }
+        ],
+        datastream: [
+            "event_log|application/n-triples|event_log.nt|nt|1431|urn:sha1:be44fc23ba7da95ba3ad67121144efc8224448ad||",
+            "md_descriptive_history|text/xml|||5209|urn:sha1:6082b819133f6be40326bcfe5fb73f4c3cc35da6||",
+            "md_descriptive|text/xml|md_descriptive.xml|xml|3882|urn:sha1:e4c1581e4f978e74382c904a706a34e5942918fa||"
+        ],
+        ancestorPath: [
+            {
+                id: "collections",
+                title: "collections"
+            }
+        ],
+        permissions: [
+            "markForDeletionUnit",
+            "move",
+            "reindex",
+            "editResourceType",
+            "destroyUnit",
+            "bulkUpdateDescription",
+            "changePatronAccess",
+            "runEnhancements",
+            "createAdminUnit",
+            "ingest",
+            "orderMembers",
+            "viewOriginal",
+            "viewAccessCopies",
+            "viewHidden",
+            "assignStaffRoles",
+            "viewMetadata",
+            "markForDeletion",
+            "editDescription",
+            "createCollection"
+        ],
+        groupRoleMap: {
+            everyone: "canViewMetadata",
+            authenticated: "canViewMetadata"
+        },
+        id: "353ee09f-a4ed-461e-a436-18a1bee77b01",
+        updated: "2023-03-16T13:28:04.959Z",
+        timestamp: 1678973288794
+    },
+    markedForDeletion: false,
+    resourceType: "AdminUnit"
+}
+
+let wrapper, router;
+
+describe('permissionUtils', () => {
+    const i18n = createI18n({
+        locale: 'en',
+        fallbackLocale: 'en',
+        messages: translations
+    });
+
+    beforeEach(() => {
+        router = createRouter({
+            history: createWebHistory(process.env.BASE_URL),
+            routes: [
+                {
+                    path: '/record/:uuid',
+                    name: 'displayRecords',
+                    component: displayWrapper
+                }
+            ]
+        });
+
+        wrapper = mount(adminUnit, {
+            global: {
+                plugins: [i18n, router],
+                stubs: {
+                    RouterLink: RouterLinkStub
+                }
+            },
+            props: {
+                recordData: recordData
+            }
+        });
+    });
+
+    it("checks for groups", () => {
```

Comment:
up to you, but if its not expensive to run more tests here it might help with readability/make expectations clearer to split this into 3 tests, 1 for each definition of the groupRoleMap. Same with the group roles and permissions tests.